templates/vue/src/components/AudioRecorder.vue

```diff
@@ -259,9 +258,5 @@ export default {
   i {
     margin: auto;
   }
-
-  .teen-status {
-    display: block;
-  }
```

Comment:
Just double-checking... Should this be deleted? It's not used?

Reply:
It's moved to `Tyde.scss`.